.circleci/config.yml

```diff
@@ -127,8 +141,45 @@ jobs:
             git checkout master
             export GITHUB_TOKEN=${CLOUD_MTA_BOT_GITHUB_TOKEN}
             curl -sL https://git.io/goreleaser | bash
-            
-  publish:
+
+  publish-to-dockerhub:
+    docker:
+      - image: cimg/go:1.17
+    working_directory: ~/go/src/github.com/SAP/cloud-mta-build-tool
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 20.10.6
+      - run:
+          name: build image pre-setup
+          command: |
+            #Make sure HEAD points to master
+            git checkout master
+            git fetch
+            git rebase
+      - run:
+          name: "build Java 17 & Node 16 image"
+          command: |
+              MBT_VERSION=$(cat ./VERSION)
+              sh $PWD/scripts/build_image 17 16.17.1 ${MBT_VERSION}
+      - run:
+          name: publish image
+          command: |
+            #Make sure HEAD points to master
+            git checkout master
+            git fetch
+            git rebase
+            MBT_VERSION=$(cat ./VERSION)
+            echo "Image release: ${MBT_VERSION}"
+            #Push to Docker Hub
+            echo "$DOCKER_HUB_TOKEN" | docker login --username $DOCKER_HUB_USER --password-stdin
+            sh $PWD/scripts/publish_image 17  16.17.1  ${MBT_VERSION} "devxci"
```

Comment:
latest node 16.x is 16.18.0 (16.17.1 also appears in other places in this file)

Reply:
you are right, node 16.18.0 published on 12-Oct-2022, when write the commit code, 16.17.1 is the latest version of node16